user/src/components/MovieDetail/Modal.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useState } from 'react';
+import './modal.scss'
+
+const Modal = ({ open, onClose ,  movie }) => {
+  if (!open) return null;
+
+  return (
+    <div onClick={onClose} className='overlay'>
```

Comment:
```suggestion
    <div onClick={onClose} className="overlay">
```
dùng nháy " cho đồng bộ e